ray-operator/controllers/ray/raycluster_controller.go

```diff
@@ -1138,6 +1157,16 @@ func (r *RayClusterReconciler) buildWorkerPod(ctx context.Context, instance rayv
 	return pod
 }
 
+func createPreStopLifecycle(preStopCommand string) *corev1.Lifecycle {
+	return &corev1.Lifecycle{
+		PreStop: &corev1.LifecycleHandler{
+			Exec: &corev1.ExecAction{
+				Command: []string{"/bin/sh", "-c", preStopCommand},
```

Comment:
```suggestion
				Command: []string{"/bin/bash", "-c", preStopCommand},
```

I think it's better to use `bash`.

Reply:
went with @rueian suggestion to let the user create the list